02Lesson/02Homework/index.html

```diff
@@ -1,60 +1,64 @@
+            <li>Frictionless Context Switching. Create a branch to try out an idea, commit a few times, switch back to where you branched from, apply a patch, switch back to where you are experimenting, and merge it in.</li>
+            <li>Role-Based Codelines. Have a branch that always contains only what goes to production, another that you merge work into for testing, and several smaller ones for day to day work.</li>
+            <li>Feature Based Workflow. Create new branches for each new feature you're working on so you can seamlessly switch back and forth between them, then delete each branch when that feature gets merged into your main line.</li>
+            <li>Disposable Experimentation. Create a branch to experiment in, realize it's not going to work, and just delete it - abandoning the work—with nobody else ever seeing it (even if you've pushed other branches in the meantime).</li>
+          </ul>
+        </article>
+        <article>
+          <h3>Community</h3>
+          <p>Questions or comments for the Git community can be sent to the mailing list by using the email address git@vger.kernel.org. Bug reports should be sent to this mailing list.</p>
+          <p>You do not need to subscribe: you will be Cc'd in replies. Please keep the Cc list intact when replying (use "Reply to all"). Greylisting may delay your first post for a few hours. Note also that the list only accepts plain-text email; please disable HTML in your outgoing messages.</p>
+          <p>The archive can be found on Gmane. Click here to subscribe. If you have trouble sending to the list with your regular mail reader, you may also post via gmane. There is also Git user mailing list on Google Groups which is a nice place for beginners to ask about anything.</p>
+        </article>
+      </section>
+      <aside>
+        <div>
+          <h4>Rapid development with FTS</h4>
+          <p>An enterprise-grade server for teams to share code, track work, and ship software — for any language, all in a single package.
+            It’s the perfect complement to your IDE.</p>
+            <img src="http://www.discountasp.net/tfs/images/tfs-2012-metro.png" alt="logo">
         </div>
         <div>
-          <h4>Node.js®</h4>
-          <p>It is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, npm, is the largest ecosystem of open source libraries in the world.</p>
-          <img src="http://nodejs-cloud.com/img/128px/nodejs.png">
+            <h4>Node.js®</h4>
+            <p>It is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, npm, is the largest ecosystem of open source libraries in the world.
+            </p>
+            <img src="http://nodejs-cloud.com/img/128px/nodejs.png" alt="logo">

```

Comment:
Looks like you hit tab one two many times here - these elements should be indented just once below their parent `div` container